### PR TITLE
docs(GBB-46): Improve syntax error response

### PR DESCRIPTION
**Pull request by PhamVanTien1910**

### CODE REVIEW

File: docs/openapi.yaml [hunk 1]
```diff
@@ -131,7 +138,17 @@ paths:
...
                 error_104_example:
-                  $ref: "#/components/examples/error_104_example"
+                  value:
+                    error_code: 104
+                    message: Invalid syntax
+                    details:
+                      [
+                        "Missing required fields",
+                        "Invalid email field format",
+                        "Invalid full_name field format",
```
Comment (hungpham28) [inline docs/openapi.yaml:148]: Format of password and full_name is what??

File: docs/openapi.yaml [hunk 2]
```diff
@@ -203,7 +227,10 @@ paths:
             application/json:
               examples:
                 error_104_example:
-                  $ref: "#/components/examples/error_104_example"
+                  value:
+                    error_code: 104
+                    message: Invalid syntax
+                    details: [The provided user ID is invalid]
```
Comment (hungpham28) [inline docs/openapi.yaml]: invalid is not found or invalid format (ex: id > 0 ) ??

File: docs/openapi.yaml [hunk 3]
```diff
@@ -173,7 +190,14 @@ paths:
             application/json:
               examples:
                 error_104_example:
-                  $ref: "#/components/examples/error_104_example"
+                  value:
+                    error_code: 104
+                    message: Invalid syntax
+                    details:
+                      [
+                        Invalid parameter passed to page,
```
Comment (hungpham28) [inline docs/openapi.yaml]: why invalid ? please more detail

File: docs/openapi.yaml (hunk 1 above)
Comment (PhamVanTien1910) [reply docs/openapi.yaml:148]: e có để regex á a

File: docs/openapi.yaml (hunk 2 above)
Comment (PhamVanTien1910) [reply docs/openapi.yaml]: đã update

File: docs/openapi.yaml (hunk 3 above)
Comment (PhamVanTien1910) [reply docs/openapi.yaml]: đã update

**Pull request merged**